net/http/inc/LinkDef.h

```diff
@@ -9,6 +9,7 @@
 #pragma link C++ class THttpWSHandler;
 #pragma link C++ class TFastCgi;
 #pragma link C++ class TCivetweb;
+#pragma link C++ class TCivetwebSSL;
```

Comment:
Do we need to expose the Civet classes? Ideally:
- they are only included by *source* files
- even their headers can move to net/http/src/
- they don't need to have dictionaries generated for them (i.e. can be removed from Linkdef.h)
Please consider this for a subsequent PR!

Reply:
Yes, I place here generic code, which can be used to plug any other http engines from external libraries. But for civetweb and fastcgi one can directly use constructors. I will implement it in following PR.

Reply:
Here: https://github.com/linev/root/commit/b9465ffc20e9cadd6d95dba4fdf417f77ca924b7